src/main/scala/com/lightbend/lagom/spec/model/Type.scala

```diff
@@ -0,0 +1,22 @@
+package com.lightbend.lagom.spec.model
+
+sealed trait Type
```

Comment:
This sealed trait is used to model types in the intermediate representation (not swagger, not lagom)